Examples/Sources/MacroExamples/Playground/PeerMacrosPlayground.swift

```diff
@@ -46,4 +46,15 @@ func runPeerMacrosPlayground() {
   MyClass().f(a: 1, for: "hello", 3.14159) { result in
     print("Eventually received \(result + "!")")
   }
+
+  // MARK: - Peer Value With Suffix Name
+
+  @PeerValueWithSuffixName
+  actor Counter {
+    var value = 0
+  }
+
+  let counter = Counter()
+
+  //  print("Peer value with suffix name for \(Counter.self): \(String(describing: Counter_peer))")
```

Comment:
Do you have any idea why `Counter_peer` con not be accessed here?

Reply:
OK, looks like it might be intentional that peer macro can’t introduce declarations on a local scope. If so, the error message should be improved. Filed https://github.com/apple/swift/issues/68661

Reply:
Thanks Alex